framework/src/Volo.Abp.TextTemplating.Abstractions/Volo/Abp/TextTemplating/TemplateDefinition.cs

```diff
@@ -37,6 +37,9 @@ public ILocalizableString DisplayName
         [CanBeNull]
         public string DefaultCultureName { get; }
 
+        [CanBeNull]
+        public string RenderEngine { get; protected set; }
```

Comment:
Making the setter public would be fine if someone want to override the template definition with a different rendering engine.